.github/workflows/nightly_publish.yml

```diff
@@ -133,15 +133,15 @@ jobs:
     if: github.repository == 'deepjavalibrary/djl'
     runs-on: [ self-hosted, gpu ]
     container:
-      image: nvidia/cuda:12.1.1-cudnn8-runtime-ubuntu20.04
+      image: nvidia/cuda:12.4.1-cudnn-devel-ubuntu22.04
```

Comment:
do we need the development image?

Reply:
Yes, TF need nvcc. If the TF is not compiled with cuda arch, it will compile on the fly.